addons/l10n_in_withholding/models/account_move.py

```diff
@@ -0,0 +1,52 @@
+from odoo import api, models, fields
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    l10n_in_withhold_type = fields.Selection([
+        ('in_withhold', 'Purchase'),
+        ('out_withhold', 'Sale')
+    ], string="Withholding Type", copy=False)
+    l10n_in_withholding_move_id = fields.Many2one('account.move',
+        string="India Withholding Entry", copy=False, readonly=True,
+        help="Link the withholding entry to the Invoice/Bill"
+    )
+    l10n_in_withholding_line_ids = fields.One2many('account.move.line', 'move_id',
+        string='Indian Withholding Lines', readonly=True,
+        compute='_compute_l10n_in_withholding_line_ids',
+    )
+    l10n_in_withhold_move_ids = fields.One2many('account.move', 'l10n_in_withholding_move_id',
+        string="Indian Withholding Entries", copy=False
+    )
```

Comment:
```suggestion
    l10n_in_withhold_type = fields.Selection([
        ('in_withhold', "Purchase"),
        ('out_withhold', "Sale")
    ], string="Withholding Type", copy=False)
    l10n_in_withholding_move_id = fields.Many2one('account.move',
        string="India Withholding Entry", copy=False, readonly=True,
        help="Link the withholding entry to the Invoice/Bill"
    )
    l10n_in_withholding_line_ids = fields.One2many('account.move.line', 'move_id',
        string="Indian Withholding Lines", readonly=True,
        compute='_compute_l10n_in_withholding_line_ids',
    )
    l10n_in_withhold_move_ids = fields.One2many('account.move', 'l10n_in_withholding_move_id',
        string="Indian Withholding Entries", copy=False
    )
```

use `"` for a string where it's visible to the end user
same for other